src/main/scala/edu/ie3/simona/model/participant/load/profile/LoadProfileStore.scala

```diff
@@ -84,14 +85,14 @@ class LoadProfileStore private (val reader: Reader) {
     * @param loadProfile
     *   the consumer type
     * @return
-    *   the maximum load in W
+    *   the maximum load in VA
     */
   def maxPower(
       loadProfile: StandardLoadProfile
-  ): squants.Power = {
+  ): ApparentPower = {
     maxParamMap.get(loadProfile) match {
       case Some(value) =>
-        Watts(value)
+        Voltamperes(value)
```

Comment:
The load profiles are specified in Watts, so why would the maximum power be an apparent power?